lib/winable.rb

```diff
@@ -0,0 +1,11 @@
+module Winable
+
+  #for use in Game
+  def win?(team_id)
+    away_win = team_id == @away_team_id && @away_goals > @home_goals
+    home_win =  team_id == @home_team_id && @home_goals > @away_goals
+    return 1 if away_win || home_win
+    0
+  end
```

Comment:
this determines a "Win" and returns 1 if true, 0 if false. Ties return 0

Reply:
nice!